client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -99,9 +120,11 @@ export default class ReserveSubmitButton extends Vue {
         newReqModule.mutateAssumedNameOriginal()
         goToNames()
         return
+      // @ts-ignore - typescript knows setup can only === 'assumed' at this point and gives error
       case 'examine':
```

Comment:
Same as above, i think enums could help you out here. Would definitely be a good idea, since these string values are used in a few places. 
If these conditions are use app-wide i would definitely recommend starting an enums directory at @src.